### DISCUSSION TITLE
Risk of damaging the microcontroller. Badly designed circuit.

### DISCUSSION THREAD

**Discussion by rtek1000** [TG9541/stm8ef#466] [General]
Hi,

As we can see in the electronic diagram, the LED of the relay can carry out a high voltage return if it fails in the reverse voltage, probably when using a power supply greater than the value that the LED supports in the reverse polarization (generally it is only 5V ).

See details in this issue: https://github.com/mister-grumbler/w1209-firmware/issues/22

Please disclose this design error to avoid unnecessary damage.

**Comment by TG9541** [comment]
It's almost incredible that such an obvious design error has been copied hundreds of times and that maybe hundreds of thousands of W1209 boards have been sold! 

I would suggest to simply remove LED "Red" or R7 and use one of the DPs of the 7S-LED display as a status indicator.

![image](https://user-images.githubusercontent.com/5466977/227616896-50d57e1a-d812-4f25-9a34-b291e5126782.png)
